README.md

```diff
@@ -37,10 +37,9 @@ Deployment
 
 👤 **JAMES ASIBEY**
 
-- GitHub:[https://github.com/jaamanu](https://github.com/jaamanu)
-- Twitter:[@JamesAsibeyManu](https://twitter.com/JamesAsibeyManu)
-- LinkedIn:(https://www.linkedin.com/in/jamesasibeymanu)
-
+-GitHub:https://github.com/jaamanu
+-Twitter:@JamesAsibeyManu
+-LinkedIn:(https://www.linkedin.com/in/jamesasibeymanu)

```

Comment:
- [ ] Your README is descritpive but it can definitely look better than it currently does. At the moment, it is difficult for anyone to get the desired information because there are no helpful headings for each section. Please reference this [template](https://github.com/microverseinc/readme-template) and rebuild your README 👍